### PR TITLE
Add JSDoc for the parameters of most constructors.

### PR DESCRIPTION
**Pull request by yT0n1**
Also remove some unneeded parameters and replace all super-type constructor calls with a call to the injector to be more robust against changes in constructor parameters in bpmn-js

### CODE REVIEW

File: lib/draw/ChoreoRenderer.js [hunk 1]
```diff
@@ -40,17 +41,21 @@ const NON_INITIATING_OPACITY = .1725;
...
 
-  BaseRenderer.call(this, eventBus, 2000);
+  BaseRenderer.call(this, eventBus, 2000); // Because of the priority this does not work with the injector
 
   function getLabel(caption, options) {
     var text = textRenderer.createText(caption || '', options);
     svgClasses(text).add('djs-label');
     return text;
   }
-
```
Comment (jan-ladleif) [inline lib/draw/ChoreoRenderer.js]: Why no empty line?

File: lib/features/popup-menu/LoopPopupProvider.js [hunk 2]
```diff
@@ -26,7 +28,6 @@ LoopPopupProvider.prototype.getHeaderEntries = function(element) {
   const isSequential = element.businessObject.loopType === 'MultiInstanceSequential';
   const isStandard = element.businessObject.loopType === 'Standard';
   let self = this;
-
```
Comment (jan-ladleif) [inline lib/features/popup-menu/LoopPopupProvider.js]: add space

File: package.json [hunk 3]
```diff
@@ -45,7 +45,8 @@
     "mocha-test-container-support": "^0.2.0",
     "puppeteer": "^1.19.0",
     "stringify": "^5.2.0",
-    "watchify": "^3.11.1"
+    "watchify": "^3.11.1",
+    "didi": "latest"
```
Comment (jan-ladleif) [inline package.json]: Can we use a concrete version number like for the other dependencies?
And is this dependency not provided by `bpmn-js` anymore like previously?

File: lib/features/label-editing/ChoreoLabelEditingProvider.js [hunk 4]
```diff
@@ -12,23 +12,25 @@ import {
...
-  LabelEditingProvider.call(this, eventBus, canvas, directEditing,
-    modeling, resizeHandles, textRenderer);
+/**
+ *
+ * @param {Injector} injector
+ * @param {EventBus} eventBus
+ * @param {Canvas} canvas
+ * @constructor
+ */
+export default function ChoreoLabelEditingProvider(injector, eventBus, canvas) {
```
[the remark this answers is not in the document]
Comment (yT0n1) [reply lib/features/label-editing/ChoreoLabelEditingProvider.js]: nope

File: package.json (hunk 3 above)
Comment (yT0n1) [reply package.json]: Was left in there by mistake

**Pull request merged**